bootstrap.sh

```diff
@@ -0,0 +1,81 @@
+# #!/bin/bash
+sudo rm -rf /etc/blocktogether
```

Comment:
The previous version of this script was idempotent, so I could run it to provision a production instance without damaging existing state. I'd like to preserve that property. Is the rm -rf necessary under vagrant? If so, maybe we can wrap this setup script in one that is specific to vagrant?